sceptre/scipool/templates/sc-product-ec2-linux-jumpcloud-workflows.j2

```diff
@@ -93,6 +93,7 @@ Resources:
   UpdateProductVersions:
     Type: Custom::ProductProvider
     Properties:
+      ReDeploy: '{{ range(1, 10000) | random }}'  # force CFN to redeploy this resource
```

Comment:
Do you think 10000 is big enough?  Maybe use 2^31?

Reply:
yeah i think but since you're asking i'll bumped it to 1 million.